platform/linux-generic/include/odp_packet_io_internal.h

```diff
@@ -328,6 +344,11 @@ int _odp_lso_create_packets(odp_packet_t packet, const odp_packet_lso_opt_t *lso
 			    uint32_t payload_len, uint32_t left_over_len,
 			    odp_packet_t pkt_out[], int num_pkt);
 
+void _odp_pktio_send_tx_compl_event(odp_buffer_t buf, const void *user_ptr, odp_queue_t queue);
```

Comment:
This function is no longer used by other modules, so it can be a static function in `odp_packet_io.c`.